bundles/org.openhab.binding.iaqualink/src/main/java/org/openhab/binding/iaqualink/internal/api/model/Home.java

```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.iaqualink.internal.api.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * {@link Home} refers to the "Home" screen of a pool controller.
+ *
+ * @author Dan Cunningham - Initial contribution
+ *
+ */
+public class Home {
+
+    @SerializedName("status")
+    @Expose
+    private String status;
+    @SerializedName("response")
+    @Expose
+    private String response;
+    @SerializedName("system_type")
+    @Expose
+    private String systemType;
+    @SerializedName("temp_scale")
+    @Expose
+    private String tempScale;
+    @SerializedName("spa_temp")
+    @Expose
+    private String spaTemp;
+    @SerializedName("pool_temp")
+    @Expose
+    private String poolTemp;
+    @SerializedName("air_temp")
+    @Expose
+    private String airTemp;
+    @SerializedName("spa_set_point")
+    @Expose
+    private String spaSetPoint;
+    @SerializedName("pool_set_point")
+    @Expose
+    private String poolSetPoint;
+    @SerializedName("cover_pool")
+    @Expose
+    private String coverPool;
+    @SerializedName("freeze_protection")
+    @Expose
+    private String freezeProtection;
+    @SerializedName("spa_pump")
+    @Expose
+    private String spaPump;
+    @SerializedName("pool_pump")
+    @Expose
+    private String poolPump;
+    @SerializedName("spa_heater")
+    @Expose
+    private String spaHeater;
+    @SerializedName("pool_heater")
+    @Expose
+    private String poolHeater;
+    @SerializedName("solar_heater")
+    @Expose
+    private String solarHeater;
+    @SerializedName("spa_salinity")
+    @Expose
+    private String spaSalinity;
+    @SerializedName("pool_salinity")
+    @Expose
+    private String poolSalinity;
+    @SerializedName("orp")
+    @Expose
+    private String orp;
+    @SerializedName("ph")
+    @Expose
+    private String ph;
+
+    public String getStatus() {
```

Comment:
You could even consider leaving our all the getters and setters it's so much boilerplate code 

cc: @davidgraeff you convinced me ;-)

Reply:
Again, i machine generated the model classes, as writing tons of boiler plate code by hand is not my idea of a good time :-) Personally i don't have an issue with pojos.